index.html

```diff
@@ -149,7 +153,7 @@
       <form action="https://github.com/apps/doska-helper/installations/new">
         <input type="submit" value="Add private repository" />
       </form>
-
+      <input id="search_repos" placeholder='Write project name'>
```

Comment:
Can you try to create similar window for admin table as well (`tasks_table`, panel `<div id="tasks">`)?

Reply:
@dkurt , i think i did it... can u review?

Reply:
Seems like the search now is case-sensitive, can you please make it insensitive?